.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 
 **/haproxy @RigglePrime
 /modules/ssh_keys.nix @RigglePrime
-/modules/redbot @RigglePrime
-/modules/monitoring/relay-nodes.nix @RigglePrime
-/nixos_systems/haproxy_byond_global.conf @RigglePrime
-/nixos_systems/haproxy_relay_global.conf @RigglePrime
+/systems/backends/systems/tgsatan/modules/redbot.nix @RigglePrime
+/systems/backends/systems/tgsatan/modules/monitoring/relay-nodes.nix @RigglePrime
+/systems/backends/modules/haproxy_base/haproxy.conf @RigglePrime
+/systems/frontends/modules/haproxy_base/haproxy.conf @RigglePrime
```

Comment:
I don't like the name frontends and backends, this will confuse people with haproxy terminology and this being named the same thing

Reply:
I went with the names in the graph in the readme. Feel free to suggest another set of names I'm not particularly attached to it.

Reply:
Relay node and game server would be most descriptive

Reply:
Renamed to edge-node and game-server